.github/workflows/main.yml

```diff
@@ -1,25 +1,39 @@
-# This is a basic workflow to help you get started with Actions
+# This Github Action workflow:
+# - Builds the frontend and backend
+# - Comments on PRs with the Terraform plan that will be applied on merge
+# - Applies Terraform plan on relevant branches
 
-name: Build and deploy to CF
+name: ai.digital.gov
```

Comment:
Changed some names, just so it reads more cleanly in the "checks" section on PRs